app.py

```diff
@@ -580,6 +614,8 @@ def get_book_info():
         book_theme=book_theme,
         book_cover=book_cover,
         book_title=book_title,
+        review=review,
+        num_review=num_review,
```

Comment:
I'm not sure how adding these extra two lines into the render_template will affect how the bookpage.html page will load from all other html pages. I'm worried this may throw errors due to the comments/reviews functionality not being fully implemented yet.